music21/metadata/__init__.py

```diff
@@ -858,10 +1361,6 @@ def fileFormat(self) -> t.Optional[str]:
         if self.fileInfo.format:
             return str(self.fileInfo.format)
 
-    @fileFormat.setter
-    def fileFormat(self, value: t.Union[str, Text]) -> None:
-        self.fileInfo.format = Text(value)
```

Comment:
has this been transferred over?

Reply:
No, the code just moved.  Here's the problem: `__settattr__` is rude.  If you implement it, none of your property setters will ever be called again.  (`__getattr__` is not rude like that).  So I had to move this code into `__setattr__`.  You'll notice that all the property setters are gone (e.g. `@composer.setter` et al).